Dockerfile

```diff
@@ -35,4 +35,4 @@ RUN apt-get update -y && \
 USER jenkins
 WORKDIR /home/jenkins
 
-CMD ["./run.sh"]
+ENTRYPOINT ["./run.sh"]
```

Comment:
Did we need to change this to a docker `ENTRYPOINT`? `CMD` generally behaves the same way and is more easily overridden. Just curious.